src/repositories/rest-user.test.ts

```diff
@@ -0,0 +1,24 @@
+import { mocked } from 'ts-jest/utils';
+import fetch, { Response } from 'node-fetch';
+import { RestUserRepository } from './rest-user';
+
+jest.mock('node-fetch');
+const mockedFetch = mocked(fetch, true);
+
+describe('REST User Repository', () => {
+    it('get current user', async () => {
+        const mockFetchPromise = {
+            json: () => Promise.resolve({ value: 3 }),
+        } as unknown;
+
+        mockedFetch.mockResolvedValue(Promise.resolve(mockFetchPromise as Response));
+
+        const repo = new RestUserRepository('user-url');
+        const user = await repo.getCurrentUser('header');
+        expect(user).toMatchObject({ value: 3 });
```

Comment:
why 3?

Reply:
Why not? :) I guess it would make more sense to make it look like the object we get back from continuum? Do you have that topology?

Reply:
According to continuum adaptor https://github.com/libero/continuum-adaptor/blob/master/src/jwt.ts#L10

Reply:
We should really come up with some standard test data that we use across the board. Doesn't have to be serious though :unicorn:

Reply:
Makes sense. In which case, address in the future as part of that task